README.md

```diff
@@ -1,70 +1,153 @@
-# Getting Started with Create React App
```

Comment:
This README is very well put together, well done. 
One thing maybe would be worth mentioning is the Facebook login, your app ID wont work for anyone else, anyone who clones your repo and wants to use your app needs to have their own facebook dev account with their own app id, maybe would be worth to put this in your README, how to get the id, and where to change it in the app.